extensions/amp-accordion/0.1/amp-accordion.css

```diff
@@ -29,3 +29,13 @@
   padding-right: 20px;
   border: solid 1px #dfdfdf;
 }
+
+amp-accordion > section:not([expanded]) > :last-child[rendersubtree] {
+  display: block !important;
```

Comment:
Please try with `display:flex` and `display:inline`. I actually don't think we need to override `display` here.

Reply:
`display:flex` works but `display: inline` does not -- the latter conflicts with the specified `width` and `height`. Since `display: none` is present in [`ampshared.css`](https://github.com/ampproject/amphtml/blob/master/css/ampshared.css#L451-L454), the property should be overridden, and since the expanded version today uses `display: block`, I'd say we should keep this as is.

Reply:
What if we were to expand the old:

```
amp-accordion > section > :last-child {
  display: none !important;
}
```

With:
```
amp-accordion > section:not([expanded]) > :last-child {
  display: none !important;
}
```
?

Reply:
That recommendation still requires an override, however the following works:
```
amp-accordion > section > :last-child:not([rendersubtree]) {
  display: none !important;
}
```

Reply:
Wait. Why do we do `display:none` by default? Wouldn't that force us to set `display: block` later?

Reply:
Yes that's right. [amp-accordion today ](https://github.com/ampproject/amphtml/blob/master/css/ampshared.css#L451-L459) is `display: none` when collapsed and `display: block` when expanded. So the original modification to use `display: block` when expanding a section is non-breaking.

Reply:
Ok. Then we have no choice :(